### ISSUE TITLE
Mismatch between map layers

### ISSUE THREAD

**Issue by madewild** [ec-doris/kohesio-backend#140]
Map overview when searching via keywords - it seems that sometimes no markers indicated on the map, only regions?
reported by @slaven777

Example:

- Go to https://kohesio.test.ec.europa.eu/projects?keywords=wifi&sort=Total-Budget-(descending)&tab=map
- Sweden is orange
- Hover it: 1 project
- Click on it > nothing

This is because the unique project has no geolocation. Known issue but bad user experience, to improve later.

**Comment by madewild** [comment]
Add disclaimer below the map?

**Comment by D063520** [comment]
Hi Max, I do not know what to do with this .... basically the projects have no geo-coordinates attached. If you want to put a disclaimer then it should be in the UI, no?

**Comment by madewild** [comment]
Yes I don't know the best way to handle this, I'll transfer to UI and postpone
